### DISCUSSION TITLE
[Done] Indicate Meteor server port on app restart

### DISCUSSION THREAD

**Discussion by msavin** [meteor/meteor#12226] [Feature Requests]
A lot of times, I have like 5 Meteor applications running. It's just easier to keep them all running than to constantly start and terminate servers. However, often times, I'll forget the port on which the apps are running.

I'm curious if anyone else runs into this situation. I thought a simple solution can be to indicate the server port on app restart

From:
```
=> Meteor server restarted
```

To:
```
=> Meteor server restarted on port 3400
```

**Comment by radekmie** [comment]
If anything, I'd show the entire `ROOT_URL` there - I often use it to use the app on another machine within the local network.

**Comment by StorytellerCZ** [comment]
I think while at it on start we should also open the browser with the app like we see elsewhere.

**Reply by msavin** [reply]
I think that can get really annoying if you're running micro-services / connected Meteor apps / re-starting server / etc. Plus I can see issues with selecting the right browser. Maybe an --open flag can be better suited for this

**Reply by Grubba27** [reply]
I would make this --open another issue/ feature but currently I have no Idea how this could be implemented. Will take a look how they do it in Angular and other frameworks

**Comment by Grubba27** [comment]
I've created the issue #12230 , if there is anything is wrong there or should be updated I would gladly do it :) this one looks like a really cool one for Hacktoberfest
